sql-plugin/src/main/spark333/scala/com/nvidia/spark/rapids/shims/ParquetLegacyNanoAsLongShims.scala

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2023, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
so I think this file could go away if we made the 332 file have this version

Reply:
I remove this file and add `{"spark": "333"}` in the ParquetLegacyNanoAsLongShims 332 file.
After running `mvn install -DskipTests -Dbuildver=333`, I receive an error:
error file=/home/cindyj/Desktop/spark-rapids/sql-plugin/target/spark333/generated/src/main/scala/com/nvidia/spark/rapids/shims/ParquetLegacyNanoAsLongShims.scala message=/home/cindyj/Desktop/spark-rapids/sql-plugin/target/spark333/generated/src/main/scala/com/nvidia/spark/rapids/shims/ParquetLegacyNanoAsLongShims.scala (No such file or directory)
Is there a way to work around this?